modules/local/run_alphafold2_pred/main.nf

```diff
@@ -30,11 +30,11 @@ process RUN_ALPHAFOLD2_PRED {
 
     output:
     path ("${fasta.baseName}*")
-    tuple val(meta), path ("${meta.id}_alphafold2.pdb"), emit: main_pdb
+    tuple val(meta), path ("${meta.id}_alphafold2.pdb")   , emit: top_ranked_pdb
```

Comment:
should we use here the prefix instead of meta id directly? So that the user could customize

Reply:
Good spot! However I am not sure if we are using it to create the reports, will check and if not will update in a new PR 😄